db/migrate/20220519094533_add_language_preferences_to_protocol_subscription.rb

```diff
@@ -0,0 +1,6 @@
+class AddLanguagePreferencesToProtocolSubscription < ActiveRecord::Migration[6.1]
+  def change
+    add_column :protocol_subscriptions, :needs_language_input, :boolean, null: false, default: false
```

Comment:
The thing that seems weird to me is why not define the `needs_language_input` on the `protocol` level, rather than on the `protocol_subscription` level? I agree that `has_language_input` should be on the `protocol_subscription` level, but surely you'd just want say "all subscriptions for this protocol should first pick their language", rather than setting it for each new protocol subscription?

What am I missing here?